servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java

```diff
@@ -111,6 +111,9 @@ public LRFUEvictor(BlockMetadataManagerView view, Allocator allocator) {
    */
   private StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocation location,
       EvictionPlan plan, List<Map.Entry<Long, Double>> sortedCRF) {
+    Preconditions.checkNotNull(location);
+    Preconditions.checkNotNull(plan);
+    Preconditions.checkNotNull(sortedCRF);
```

Comment:
What's the convention for precondition checks? Should we still do it even if the values are isolated from user input?